IPython/terminal/debugger.py

```diff
@@ -131,6 +132,18 @@ def in_thread():
         except Exception:
             raise
 
+    def do_interact(self, arg):
+        ipshell = embed.InteractiveShellEmbed(
+            config=self.shell.config,
+            banner1="*interactive*",
+            exit_msg="now exiting InteractiveConsole...",
```

Comment:
Maybe this one small change:

```suggestion
            exit_msg="*exiting interaractive...*",
```